yarn.lock

```diff
@@ -2588,6 +2588,11 @@ js-tokens@^3.0.0, js-tokens@^3.0.2:
   version "3.0.2"
   resolved "https://registry.yarnpkg.com/js-tokens/-/js-tokens-3.0.2.tgz#9866df395102130e38f7f996bceb65443209c25b"
 
+"js-tokens@^3.0.0 || ^4.0.0":
```

Comment:
I ran a `yarn install` to fix this issue:

https://travis-ci.org/github/davidkpiano/react-redux-form/jobs/666598346#L219

I don't think the package.json/yarn.json need to be updated; it seems like maybe the lock-files were just outdated?